src/components/UpdateBook.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { FC } from 'react';
+import { supabase } from '../utils/supabase';
+import { IBook } from '../interfaces/IBook.interface';
+
+interface IBookProps {
+  id?: string;
+  quantity?: number;
+}
+
+const AddBook: FC<IBookProps> = ({ id, quantity }) => {
```

Comment:
Did you mean "UpdateBook"? 
```suggestion
const UpdateBook: FC<IBookProps> = ({ id, quantity }) => {
```